Production/crab/submit.py

```diff
@@ -9,20 +9,26 @@
 
```

Comment:
- submit.py should be split into 2 files as in TauTriggerTools to be compatible with the latest crab version
- I suggest copy-paste 3 files from TauTriggerTools repository and put them in Production/scripts and Production/python folders respectively.
   - although some minimal modifications will still be required:
      - fix paths
      - add the possibility to store in abs path (e.g. /store/group/phys_tau/) instead of relative /store/user/USER_NAME